src/Umbraco.Web/Models/ContentEditing/CodeFileDisplay.cs

```diff
@@ -14,6 +14,9 @@ public class CodeFileDisplay : INotificationModel
         [DataMember(Name = "virtualPath", IsRequired = true)]
         public string VirtualPath { get; set; }
 
+        [DataMember(Name = "path", IsRequired = true)]
+        public string Path { get; set; }
```

Comment:
Also, IsRequired doesn't need to be true here, this is actually a readonly field right? Should be decorated with ReadOnly attribute